tests/tests.rs

```diff
@@ -45,5 +47,56 @@ async fn fairing_init_with_specific_traces_sampler() {
         .await
         .expect("Rocket failed to ignite");
 
-    assert!(hub.client().is_some());
+    let sentry_client = hub.client().unwrap();
+    assert!(sentry_client.options().traces_sampler.is_some());
+}
+
+#[rocket::async_test]
+async fn fairing_init_with_debug_rocket_profile() {
+    let figment = Figment::from(Config::debug_default()).join(SENTRY_DSN_CONFIG);
+    let _rocket = rocket::custom(figment)
+        .attach(RocketSentry::fairing())
+        .ignite()
+        .await
+        .expect("Rocket failed to ignite");
+    let client = Hub::current().client().unwrap();
+    let client_options = client.options();
+    let environment = client_options.environment.clone().unwrap().to_string();
```

Comment:
A lot of duplication between these tests. Let's create a helper function if it's not too much effort.

Reply:
I tried something, let me know if that's what you had in mind

Reply:
Yes, very good :+1: